### ISSUE TITLE
Download link is busted for exe

### ISSUE THREAD

**Issue by chronocide** [scottrice/Ice#466]
The main download link (http://consolegrid.com/ice.zip) is not working.  Looks like the domain expired or something.

**Comment by vereis** [comment]
Yeah, you can probably still grab the download from [archive.org](https://web.archive.org/web/20151028164620/http://consolegrid.com/ice.zip)

**Comment by anewuser** [comment]
https://github.com/scottrice/Ice/releases
